### ISSUE TITLE
Initial Plan

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/OpenImagesObjectDetection2019#1]
Here's what I think we should do:

### Initial Submission 

- [x] Download train/valid/test sets
- [ ] Train RetinaNet for a single epoch
- [ ] Run against validation set
- [x] Get per-class metrics
- [ ] Run against test set
- [ ] Submit so we can get Google Cloud credits

### Investigation and Iteration
- [ ] Create a dataset with ~10 balanced classes 

### EDA
- [x] Figure out class balance
- [ ] Create a balanced dataset

### Misc
 - [ ] Investigate custom loss function?
